Dockerfile.centos.7_9.x

```diff
@@ -1,5 +1,4 @@
 FROM centos:centos7
-MAINTAINER Jean Baptiste Favre <docker@jbfavre.org>
```

Comment:
you dare calling this a minor changes ?
How does "stealing other's work" looks like to you ?

Reply:
I assure you this was not our intention.
How best can we resolve this?
I can certainly put you back as MAINTAINER in our fork... it just didn't seem appropriate for our developers to contact you for something we plan to maintain ourselves.